Packs/FTP/pack_metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "FTP",
+    "description": "FTP integration to download or upload file to remote ftp server. Please be noted that FTP transfer is insecure. Please use it with care. ",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Jie Liau",
+    "url": "",
+    "email": "",
+    "created": "2024-06-01T06:43:37Z",
+    "categories": [],
```

Comment:
In which category this pack is from the following list of possible categories: Analytics & SIEM, Case Management, Cloud Services, Data Enrichment & Threat Intelligence, Database, Deception & Breach Simulation, Endpoint, Forensics & Malware Analysis, IT Services, Identity and Access Management, Messaging and Conferencing, Network Security, Utilities, Vulnerability Management, Authentication & Identity Management, Email?